lib/IRGen/GenExistential.cpp

```diff
@@ -1858,8 +1858,11 @@ OwnedAddress irgen::emitBoxedExistentialContainerAllocation(IRGenFunction &IGF,
 
   auto &destTI = IGF.getTypeInfo(destType).as<ErrorExistentialTypeInfo>();
   auto srcMetadata = IGF.emitTypeMetadataRef(formalSrcType);
-  // Should only be one conformance, for the Error protocol.
-  assert(conformances.size() == 1 && destTI.getStoredProtocols().size() == 1);
+  // Should only be one stored conformance, for the Error protocol.
+  // But there are 1+NumInvertibleProtocols conformances in the existential's
+  // layout.
+  assert(conformances.size() == 1+NumInvertibleProtocols);
```

Comment:
I feel like this is a bug because Error's requirement signature should have `Self: Copyable`. So the minimal signature for `<T where T: Copyable, T: Error>` is just `<T where T: Error>`, and ExistentialLayout::getProtocols() should then just return `Error`.

Reply:
I've fixed this so that it just has one protocol, Error, in the layout, since Copyable is implied.